gigadb/app/tools/wasabi-migration/docker-compose.yml

```diff
@@ -15,10 +15,14 @@ services:
     volumes:
       - ./config:/root/.config/rclone
       - .:/app
-        
+    command: /app/spin_up_swatchdog.sh
```

Comment:
It feels confusing to use the `rclone` service to create the Swatchdog container and start the tool. I think it's better to have a specific Swatchdog Docker service called something like `rclone_monitor` to be defined in `docker-compose.yml`. You could even think about starting the Swatchdog service in `docker-compose.yml` using something similar to below:
```
command: swatchdog -c /app/config/swatchdog.conf -r $RESTART_TIME -t "$$(cat /app/logs/latest_rclone_log)"
```